pwndbg/commands/heap.py

```diff
@@ -704,10 +717,33 @@ def vis_heap_chunks(addr=None, count=None, naive=None):
 
     cursor = cursor_backup
 
+    has_huge_chunk = False
+    # round up to align with 4*ptr_size and get half
+    half_max_size = ((max_visualize_chunk_size - 1) // (ptr_size << 2) + 1) * (ptr_size << 2) // 2
+
     for c, stop in enumerate(chunk_delims):
         color_func = color_funcs[c % len(color_funcs)]
 
+        if stop - cursor > 0x10000:
+            has_huge_chunk = True
+        first_cut = True
+        # round down to align with 2*ptr_size
+        begin_addr = cursor // (ptr_size << 1) * (ptr_size << 1)
```

Comment:
We can use `pwndbg.lib.memory.align_down` or `pwndbg.lib.memory.round_down` here, and in the line below.

Reply:
The `round_up` and `round_down` should work.